app/Models/SocialAccount.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Concerns\HasUuids;
+use Illuminate\Support\Str;
+
+class SocialAccount extends Model
+{
+    use HasFactory, HasUuids;
+
```

Comment:
Should define const for fillable column and reuse

```suggestion
const USER_ID = 'user_id';
...
```